Cargo.toml

```diff
@@ -19,6 +19,7 @@ crossbeam = { version = "0.8.2", optional = true }
 crossterm = { version = "0.27.0", features = ["serde"] }
 fd-lock = "3.0.3"
 itertools = "0.10.3"
+nix = { version = "0.27.1", features = ["term"] }
```

Comment:
Not super critical but we could make this a unix only dependency for slightly faster downloads on windows.

Reply:
Thanks, forgot to do that.